purescript.cabal

```diff
@@ -79,7 +79,16 @@ library
                    process >= 1.2.0 && < 1.5,
                    safe >= 0.3.9 && < 0.4,
                    semigroups >= 0.16.2 && < 0.19,
-                   parallel >= 3.2 && < 3.3
+                   parallel >= 3.2 && < 3.3,
+                   stm,
+                   regex-tdfa,
+                   edit-distance,
```

Comment:
Could you please rerun `license-generator` and make sure everything is compatible with MIT?